polkadot/runtime/parachains/src/mock.rs

```diff
@@ -411,6 +418,13 @@ parameter_types! {
 	pub MaxXcmTransactWeight: Weight = Weight::from_parts(10_000_000, 10_000);
 }
 
+pub struct BrokerPot;
+impl Get<InteriorLocation> for BrokerPot {
+	fn get() -> InteriorLocation {
+		todo!()
```

Comment:
Needs doing?

Reply:
Not really, it's never called. Will change it to `unimplemented`.